### DISCUSSION TITLE
Can jaspr be used in a flutter client as an html editor for an end user

### DISCUSSION THREAD

**Discussion by graemep-nz** [schultek/jaspr#261] [General]
Since jaspr can generate html, is it possible it could be used in an html editor in a flutter client app similar to 
https://pub.dev/packages/html_editor_enhanced
so that an end user can generate html (like rich text)

**Comment by schultek** [comment]
It's definitely possible, but probably not trivial.

Jaspr is made to always render whole documents, not just html snippets. So you'd have to go into the lower level apis to achieve that (basically create your own AppBinding).

**Reply by graemep-nz** [reply]
ok, so if a jaspr website needed an html editor, would it be able to use html_editor_enhanced - since it's dart code?

**Reply by schultek** [reply]
No, since its a flutter widget
